Formula/cayley.rb

```diff
@@ -84,6 +86,15 @@ def plist; <<~EOS
   end
 
   test do
+    # Check version
     assert_match version.to_s, shell_output("#{bin}/cayley version")
+
+    # Check UI
+    fork do
+      exec "#{bin}/cayley", "http"
```

Comment:
if you can specify the port, please do using `free_port` like https://github.com/Homebrew/homebrew-core/blob/master/Formula/traefik.rb#L61